website/views.py

```diff
@@ -30,6 +30,12 @@ def get_context_data(self, **kwargs):
                     kwargs['partner_entry'] = ordered_entry.entry
             except EntryCollection.DoesNotExist:
                 pass
+
+        # Zip the articles together with the lengths that each preview should be truncated to.
+        # This also takes care of limiting the number of articles to a max of 4.
+        if kwargs['featured_entry_set']:
```

Comment:
The more I think about this, the more I think the article excerpt length should exist on the template and not the view, just so that the client can adjust if/when they see fit.